src/tests/ftest/daos_test/dfuse.py

```diff
@@ -79,6 +79,10 @@ def test_daos_dfuse_unit(self):
 
         cmd = [self.daos_test, '--test-dir', mount_dir]
 
+        # Metadata tests don't work with caching enabled
+        if intercept or cache_mode == 'nocache':
+            cmd.append('--metadata')
```

Comment:
The issue is with writeback cache where the kernel assumes control of mtime and size and from memory this test skips the interception tests when using writeback already.

Reply:
If you run all variants, you can see almost all of the caching ones fail the mtime tests. I opened https://daosio.atlassian.net/browse/DAOS-11204 for this, but it was closed as duplicate / won't fix

Reply:
That might have been a mistake, it was closed as a dupliate of a ticket that was closed because it was based on writeback caching.